tests/storage/upgrade/test_upgrade_storage.py

```diff
@@ -164,19 +187,30 @@ def test_vm_snapshot_restore_check_after_upgrade(
         ],
         scope=DEPENDENCY_SCOPE_SESSION,
     )
-    def test_vm_snapshot_restore_create_after_upgrade(self, cirros_vm_for_upgrade_b, snapshots_for_upgrade_b):
+    def test_vm_snapshot_restore_create_after_upgrade(self, rhel_vm_for_upgrade_b, snapshots_for_upgrade_b):
         with VirtualMachineRestore(
-            name=f"restore-snapshot-{cirros_vm_for_upgrade_b.name}",
+            name=f"restore-snapshot-{rhel_vm_for_upgrade_b.name}",
             namespace=snapshots_for_upgrade_b.namespace,
-            vm_name=cirros_vm_for_upgrade_b.name,
+            vm_name=rhel_vm_for_upgrade_b.name,
             snapshot_name=snapshots_for_upgrade_b.name,
         ) as vm_restore:
+            rhel_vm_for_upgrade_b.stop(wait=True)
```

Comment:
We need to check if it's ready first (otherwise stop call may fail)
```suggestion
        if rhel_vm_for_upgrade_a.ready:
            rhel_vm_for_upgrade_a.stop(wait=True)
```

Reply:
Done in next commit!